src/_includes/components/macros.njk

```diff
@@ -1,4 +1,4 @@
-{% macro applies_to(applies, type='info') %} {# body, title='', type='info') #}
```

Comment:
These dashes remove extra whitespace, which is less important for these block-level template functions but essential for inline ones like glossary links, etc